contracts/src/base/registry/facets/delegation/SpaceDelegationFacet.sol

```diff
@@ -186,6 +224,10 @@ contract SpaceDelegationFacet is ISpaceDelegation, OwnableBase, Facet {
   }
 
   function _isValidSpaceOwner(address space) internal view returns (bool) {
-    return IERC173(space).owner() == msg.sender;
+    address spaceFactory = SpaceDelegationStorage.layout().spaceFactory;
+
+    return
+      IERC173(space).owner() == msg.sender &&
+      IArchitect(spaceFactory).getTokenIdBySpace(space) > 0;
```

Comment:
Is the token ID of the first space 0 or 1?

Reply:
first space is a test space no owner.